src/OpenTelemetry/Metrics/Exemplar/Exemplar.cs

```diff
@@ -28,31 +30,149 @@ namespace OpenTelemetry.Metrics;
 #endif
     struct Exemplar
 {
+    internal HashSet<string>? KeyFilter;
```

Comment:
A better name would be "AllowedKeys" ?

Reply:
It is the opposite. `KeyFilter` is essentially the list of keys set on the view which will be part of `MetricPoint`. We want to filter those out as we enumerate and only return the tags NOT in that set. I could rename it `ExcludedKeys` if you think that is better?

Reply:
ExcludedKeys or KeysToBeExluded sounds better to me.

Reply:
How's this? https://github.com/open-telemetry/opentelemetry-dotnet/pull/5364/commits/0ffba7d6d17cfba223d77057ed764be461807735